src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1039,6 +1039,8 @@ public String DependsOnNETStandard
         [Output]
         public ITaskItem[] UnresolvedAssemblyConflicts => _unresolvedConflicts.ToArray();
 
+        public bool FailIfNotIncremental { get; set; }
```

Comment:
This should probably go above the [Output]s.

Is anyone other than us expected to use this directly? If not, we can consider just making it built-in.

Reply:
Moved.